.gitignore

```diff
@@ -1,18 +1,321 @@
-# Android
+# Created by https://www.toptal.com/developers/gitignore/api/macos,linux,gradle,windows,android,androidstudio,java,kotlin
```

Comment:
I don't think we need to add this extra configuration to the `.gitignore`. We can add new entries as needed.

Reply:
I think we can add common rules in project root `.gitIgnore`, prevent files that should be ignored added by contributors